packages/zk-certificates/issuedZkCertificates/21519148476170378733908765795258099440584106433051137421983306394866425339640.json

```diff
@@ -0,0 +1,7 @@
+  "holderCommitment": "20667153878906325333507580918310525984100181224769672762345736961745003970670"
+}
```

Comment:
These issued certificate files do not belong on git. I guess you need to update the .gitignore according to the renaming.

Reply:
updated .gitignore